src/c76_menuCAL.hpp

```diff
@@ -168,6 +168,17 @@ bool checkProgressiveUpDown(int *val, int minDelay = 25)
   return ret;
 }
 
+bool checkProgressiveUpDown(int *val, int minDelay = 25)
```

Comment:
i dont understand this overload. we allow passing an int while handling it as a long. why is it needed? also this could lead to weird/invalid memory access. Especially since the value behind it is being modified by a side effect.

Reply:
There are many functions that use this to handle ints, and one that handles long. I don't understand how this can possibly lead to weird/invalid memory access. This simply allows longs and ints to use this function and we stop an int from overflowing.

Reply:
convinced after discussion on discord.